### DISCUSSION TITLE
Only issue wildcard certs

### DISCUSSION THREAD

**Discussion by AskAlice** [onedr0p/cluster-template#727] [General]
Because letsencrypt certificates are public knowledge, it only makes sense to only utilize and issue wildcard certs to obscure the names of services to avoid getting scrubbed by security researchers constantly.

Thoughts?

**Comment by onedr0p** [comment]
This is already done, it sets up a WC certificate by default.

https://github.com/onedr0p/flux-cluster-template/blob/main/kubernetes/apps/networking/ingress-nginx/certificates/certificates.yaml

It was done back here... https://github.com/onedr0p/flux-cluster-template/discussions/592
